content_script.js

```diff
@@ -10,6 +10,11 @@ captureScreenshot = function () {
   logger("Capturing screenshot");
   var canvas = document.createElement("canvas");
   var video = document.querySelector("video");
+  if (video.mediaKeys != null) {
+    console.log("drm protected 1");
```

Comment:
Use `logger` or remove it next if only for debug.

Reply:
That code was meant for me to debug the message temporarily, I forgot to get rid of it.